mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -181,7 +188,7 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
         xlabel = "Time [years]"
         ylabel = "[x$10^{22}$ J]"
 
-        if ref_casename_v0 != "None":
+        if ref_casename_v0 != 'None':
             figname = "{}/ohc_{}_{}_{}.png".format(plots_dir,
```

Comment:
ok, I think I changed all double quotes to single quotes in the 3 timeseries scripts.